src/CosmosDB/CosmosDB/help/New-AzCosmosDBAccount.md

```diff
@@ -64,7 +64,7 @@ A new CosmosDB Account with name databaseAccountName is created in the ResourceG
 
 ### Example 2
 ```powershell
-New-AzCosmosDBAccount -ResourceGroupName resourceGroupName -Name restored-account-name  -Location "West US" -FromPointInTimeBackup -RestoreSourceId /subscriptions/{subscriptionId}/providers/Microsoft.DocumentDB/restorableDatabaseAccounts/{instance-id} -RestoreTimesampInUtc 2020-07-20T17:19:25+0000
+New-AzCosmosDBAccount -ResourceGroupName resourceGroupName -Name restored-account-name  -Location "West US" -FromPointInTimeBackup
```

Comment:
Strange, it can associate parameter name "RestoreTimestampInUtc", but it do be not in the parameters list.

Reply:
As RestoreTimestampInUtc is bounded as dynamic parameter

https://github.com/Azure/azure-powershell/blob/371a6785020ad69b78a7b79ef5ef3a72379e1456/src/CosmosDB/CosmosDB/CosmosDBAccount/NewAzCosmosDBAccount.cs#L277

https://github.com/Azure/azure-powershell/blob/864448ba83f26d6826195e0da158891183876962/src/CosmosDB/CosmosDB/CosmosDBAccount/RestoreRequestDynamicParameters.cs#L28-L35